calculator2.py

```diff
@@ -0,0 +1,13 @@
+def add(x, y):
+    return x + y
+
+def subtract(x, y):
+    return x - y
+
+def multiply(x, y):
+    return x * y
+
+def divide(x, y):
+    if y == 0:
+        raise ValueError("Cannot divide by zero!")
```

Comment:
Consider handling the case when the user tries to divide by zero more gracefully. Instead of raising a ValueError, you could return a specific message or a special value like 'inf'. [medium]